modules/cert-manager-certificate-api-server.adoc

```diff
@@ -9,7 +9,7 @@
 .Prerequisites
 
 * You have access to the cluster with `cluster-admin` privileges.
-* You have installed the {cert-manager-operator} 1.13.0 or later.
+* You have installed version 1.13.0 or later of the {cert-manager-operator}.
```

Comment:
Does the [Creating certificates for user workloads](https://82942--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-creating-certificate#cert-manager-certificate-mgmt_cert-manager-creating-certificate), in the same assembly as this, prerequisite need the Operator version?

Reply:
I'm not super familiar with these docs but I think the instances that don't mention a version are intentional. Those procedures probably just need any version of the Operator, while the ones that mention a version number are probably saying "this procedure's feature is only available in versions x+"